<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuadZone</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    background-color: #FFFFFF;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_QuadZone">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F37321; 
background-color: #FFFFFF;</string>
     </property>
     <property name="text">
      <string>QuadZone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="titleSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="signInPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#signInPage {
    background-color: #FFFFFF;
    border-radius: 15px;
    border: 1px solid #E0E0E0;
}</string>
      </property>
      <layout class="QVBoxLayout" name="signInLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label_SignInEmail">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_SignInEmail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_SignInPassword">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_SignInPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="signInSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SignIn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F37321;
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #E5651C;
}
QPushButton:pressed {
    background-color: #D4591A;
}</string>
         </property>
         <property name="text">
          <string>Sign In</string>
         </property>
        </widget>
       </item>

       <item>
        <layout class="QHBoxLayout" name="linkLayout">
         <item>
          <widget class="QPushButton" name="pushButton_ForgotPassword">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: transparent;
    color: #666666;
    border: none;
    text-decoration: underline;
}
QPushButton:hover {
    color: #F37321;
}</string>
           </property>
           <property name="text">
            <string>Forgot password?</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="linkSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_GoToRegister">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>11</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: transparent;
    color: #666666;
    border: none;
    text-decoration: underline;
}
QPushButton:hover {
    color: #F37321;
}</string>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="registerPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#registerPage {
    background-color: #FFFFFF;
    border-radius: 15px;
    border: 1px solid #E0E0E0;
}</string>
      </property>
      <layout class="QVBoxLayout" name="registerLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label_RegisterEmail">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_RegisterEmail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_RegisterPassword">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_RegisterPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="registerSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Register">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F37321;
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #E5651C;
}
QPushButton:pressed {
    background-color: #D4591A;
}</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_BackToSignIn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    color: #666666;
    border: none;
    text-decoration: underline;
}
QPushButton:hover {
    color: #F37321;
}</string>
         </property>
         <property name="text">
          <string>Back to Sign In</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resetPasswordPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#resetPasswordPage {
    background-color: #FFFFFF;
    border-radius: 15px;
    border: 1px solid #E0E0E0;
}</string>
      </property>
      <layout class="QVBoxLayout" name="resetPasswordLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label_ResetEmail">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ResetEmail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_NewPassword">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #333333;</string>
         </property>
         <property name="text">
          <string>New Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NewPassword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    background-color: #FFFFFF;
    color: #333333;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 8px 12px;
}
QLineEdit:focus {
    border: 2px solid #F37321;
    outline: none;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="resetSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="resetButtonLayout">
         <item>
          <widget class="QPushButton" name="pushButton_Cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: transparent;
    color: #666666;
    border: 1px solid #D0D0D0;
    border-radius: 8px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #F5F5F5;
    color: #333333;
}
QPushButton:pressed {
    background-color: #E0E0E0;
}</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ResetPassword">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #F37321;
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #E5651C;
}
QPushButton:pressed {
    background-color: #D4591A;
}</string>
           </property>
           <property name="text">
            <string>Reset Password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>